lms/templates/ccx/enrollment.html

```diff
@@ -26,7 +27,7 @@
   </div>
 
   <div class="enroll-option">
-    <input type="checkbox" name="email-students" value="Notify-students-by-email" checked="yes">
+    <input type="checkbox" name="email-students" id="email-students" value="Notify-students-by-email" checked="yes">
```

Comment:
@carsongee please see next like , there is a label, label requires id in **for** attribute. See http://www.w3schools.com/tags/tag_label.asp
Id was missing in this case and label was pointed to **name**. Because of this screen read was not getting label on focusing the input field.